src/python/pants/backend/kotlin/compile/kotlinc.py

```diff
@@ -197,8 +198,11 @@ async def compile_kotlin_source(
     )
     output: ClasspathEntry | None = None
     if process_result.exit_code == 0:
+        stripped_digest = await Get(
+            Digest, StripJarRequest(digest=process_result.output_digest, filenames=(output_file,))
+        )
```

Comment:
`kotlinc` produces reproducible jars by default, so it's not necessary to strip here.